server/users.ts

```diff
@@ -84,6 +85,7 @@ function add(user: User) {
 
 	if (users.has(user.userid)) throw new Error(`userid taken: ${user.userid}`);
 	users.set(user.userid, user);
+	updateForcedPublic(user);
```

Comment:
isn't the userid always going to be `guest<numbers>` here?

Reply:
OK, thats what I saw in testing, but wasn't confident. Removed.